padd.sh

```diff
@@ -654,11 +629,14 @@ ceol=$(tput el)
 # wrapper - echo with a clear eol afterwards to wipe any artifacts remaining from last print
 CleanEcho() {
   echo -e "${ceol}$1"
+  #echo -e "$1" "${ceol}"
 }
 
 # wrapper - printf
 CleanPrintf() {
 # tput el
+# don't know how to fix this!?
+# shellcheck disable=SC2059
```

Comment:
Does `printf "%s" "$@"` work? (Please test! it does satisfy the test - but is the real output as expected?)

Reply:
I'm sorry, but this screws up everything:

<img width="1534" alt="image" src="https://user-images.githubusercontent.com/18568381/159571047-9dea5661-fa15-4eb8-8c08-e5e1cb8a2ad6.png">
🤷🏻‍♂️

Reply:
Try `printf "%s/n" "$@"`

`printf` doesn't add newlines to the string.

Reply:
Nope, this looks pretty much the same 😞

Reply:
I think it's fine to disable the check here. We do pass string formating instructions within `"${@}"`

https://github.com/koalaman/shellcheck/issues/1028